packages/react-components/react-toast/src/state/vanilla/createToast.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { ToastOptions } from '../types';
+import { EVENTS } from '../constants';
+
+let counter = 0;
+
+export function createToast(content: React.ReactNode, options: ToastOptions = {}, targetDocument: Document) {
```

Comment:
it's kind of weird for me to have a function with the name `createToast` that actually doesn't return a `Toast` instance, what will you call `Toast` constructor once you convert it to closure? (I'd expect the constructor to be converted to `createToast`)

This method looks more like a `dispatchToast`?

Reply:
yeah dispatchToast sounds good